lib/pages/search/recent_item_widget.dart

```diff
@@ -167,19 +170,17 @@ class _ContactInformation extends StatelessWidget {
           ),
         ),
         if (presentationSearch.directChatMatrixID != null)
-          Text(
-          presentationSearch.directChatMatrixID ?? "",
-          overflow: TextOverflow.ellipsis,
-          maxLines: 1,
-          softWrap: false,
-          style: Theme.of(context).textTheme.bodyMedium?.merge(
-            TextStyle(
-              overflow: TextOverflow.ellipsis,
-              letterSpacing: 0.15,
-              color: LinagoraRefColors.material().tertiary[30],
+          HighlightText(
+            text: presentationSearch.directChatMatrixID ?? "",
```

Comment:
how about display name? how about the email?

Reply:
- It's information widget and Julian support highlights for this widget. If it was not fine I'll remove it. WDYT?

![Simulator Screenshot - iPhone 14 - 2023-07-31 at 12 31 10](https://github.com/linagora/twake-on-matrix/assets/99852347/4bf03eb5-c548-47c9-bd3d-9f8c7f474755)